### ISSUE TITLE
Cannot login to apple ID

### ISSUE THREAD

**Issue by SonixSpine** [powenn/AltServer-Linux-ShellScript#23]
Whenever I put in my ID it gives me the error
    Unable to sign you in to your Apple ID. Try again later. (-36607)
I don't know what it means or how to fix it.

**Comment by powenn** [comment]
> Whenever I put in my ID it gives me the error
> 
>     Unable to sign you in to your Apple ID. Try again later. (-36607)
> 
> I don't know what it means or how to fix it.

Try few more times, I think it's network problems

**Comment by SonixSpine** [comment]
Yeah I got the issue fixed, but now whenever i try it tells me that my device is already in the "Team" even though I don't have altstore installed at all.

**Comment by powenn** [comment]
Please try the latest release, this issue should be solved

**Comment by ddeler** [comment]
No luck for me, same issue as OP. I've already tried to compulsively press "Refresh All" but it keeps giving me the same error code

**Comment by LuNeder** [comment]
From [AltServer-Linux Releases](https://github.com/NyaMisty/AltServer-Linux/releases):

Many user have experienced -36607 error, which is a new Apple restriction based on Anisette Data.

To solve this, you can use your own anisette data service:

    Pull and run [alt_anisette_server](https://hub.docker.com/r/nyamisty/alt_anisette_server) according to README.

    If you cannot build the server yourself, you can also ask other people to share the same server together

    Run export ALTSERVER_ANISETTE_SERVER="http://192.168.1.2:6969" before AltServer-Linux each time (replace the 192.168.1.2 to your server IP)
    Run AltServer-Linux
